### ISSUE TITLE
Digit and analog images not saved. 

### ISSUE THREAD

**Issue by stevebeer** [jomjol/AI-on-the-edge-device#1305]
### The Problem

I would like to tune the digit and analog ROI's.
Therefore i want to use the logged images in the folders /log/digit and /log/analog.
Subdirs (timestamp) are made, but no jpg files are written.

Also folder img_tmp stays empty.
(recognition works, pretty good mqtt values are send  out)

### Version

v12.0.1

### Screenshots

![image](https://user-images.githubusercontent.com/9861915/200774915-cd29620c-0768-4622-ab1c-0273ad08b8e2.png)

**Comment by wlemkens** [comment]
I have the same issue.

**Comment by Ingo-Knito** [comment]
Is fixed in rolling. See #1189
I had the same issue in #1192

**Comment by wlemkens** [comment]
> This is strange and has never been reported. To exclude configuration problems: did you reboot after changing any setting?

I did, but just to be 100% sure and did it again. After rebooting it wasn't fixed.

 
> If so you should find the images in the subdirectory looking like this: `/log/digit/20221109/00/2.0_main_dig1_20221109-003458.jpg`

The directories are created (e.g.: /log/digit/20221109/00/), but they are all empty.

**Comment by Kimble8650** [comment]
> I have the same issue.

Same here 🙋

**Comment by jomjol** [comment]
This is strange and difficult to handle. My system is working properly and I don't know where to attack this problem?!

Current I have no clue, what is going on there !?

**Comment by stevebeer** [comment]
Hi Jomjol,

This is my config after flashing the ESP32-CAM with the online flash tool from your site. 
The SDcard (16GB) is formatted FAT32. Data for SDcard is with files from Master branch, 12.0.1

[MakeImage]
;LogImageLocation = /log/source
WaitBeforeTakingPicture = 5
;LogfileRetentionInDays = 15
Brightness = 0
Contrast = 0
Saturation = 0
LEDIntensity = 50
ImageQuality = 12
ImageSize = VGA
FixedExposure = false

[Alignment]
InitialRotate = 335
InitialMirror = false
SearchFieldX = 20
SearchFieldY = 20
AlignmentAlgo = default
FlipImageSize = false
/config/ref0.jpg 68 219
/config/ref1.jpg 386 157

[Digits]
Model = /config/dig-cont_0570_s3.tflite
CNNGoodThreshold = 0.5
;LogImageLocation = /log/digit
;LogfileRetentionInDays = 3
main.dig1 55 137 34 61 false
main.dig2 95 136 35 63 false
main.dig3 134 134 36 65 false
main.dig4 177 134 37 67 0
main.dig5 216 135 39 71 0
main.dig6 255 135 36 66 0
main.dig7 297 136 35 65 0
main.dig8 341 133 38 70 0

[Analog]
Model = /config/ana-cont_11.3.0_s2.tflite
;LogImageLocation = /log/analog
;LogfileRetentionInDays = 3
main.ana1 307 231 168 168 false

[PostProcessing]
main.DecimalShift = -3
PreValueUse = true
PreValueAgeStartup = 720
AllowNegativeRates = false
main.MaxRateValue = 0.5
;main.MaxRateType = AbsoluteChange
;main.ExtendedResolution = false
;main.IgnoreLeadingNaN = true
ErrorMessage = true
CheckDigitIncreaseConsistency = true

[MQTT]
Uri = mqtt://192.168.178.4:1883
MainTopic = /Watermeter/value
ClientID = water
user = hamqtt
password = hamqtt
;SetRetainFlag = true

;[InfluxDB]
;Uri = undefined
;Database = 
;Measurement = undefined
;user = undefined
;password = undefined

;[GPIO]
;IO0 = input disabled 10 false false 
;IO1 = input disabled 10 false false 
;IO3 = input disabled 10 false false 
;IO4 = built-in-led disabled 10 false false 
;IO12 = input-pullup disabled 10 false false 
;IO13 = input-pullup disabled 10 false false 
LEDType = WS2812
LEDNumbers = 2
LEDColor = 150 150 150

[AutoTimer]
AutoStart = true
Intervall = 3

[Debug]
Logfile = false
LogfileRetentionInDays = 3

[System]
TimeZone = CET-1CEST,M3.5.0,M10.5.0/3
;TimeServer = undefined
;AutoAdjustSummertime = false
;Hostname = undefined
SetupMode = false

**Comment by jomjol** [comment]
With this config.ini the problem is clear - the logging is disabled, which you can recognize by the `;` before the two important setting (analog as well).

```
[Digits]
...
;LogImageLocation = /log/digit
;LogfileRetentionInDays = 3
```

This is clearly not consistant with the screen shot above. Either we have a problem with the graphal configuration or there is some in consistency within your configuration. For a quick and dirty solution you can remove the `;` manually and the logging should work.

**Comment by stevebeer** [comment]
Okay, i double checked it by removing the ; uncomment signs.
I've saved the config file and rebooted. The same thing happens, nothings changes.

The reason why it was uncommented in the quoted config file was because i had copied the original files (12.0.1) from github back to the sd-card after i opened this topic. i'm sorry for the misunderstanding.

**Comment by caco3** [comment]
@stevebeer Is this still an issue or can we close the issue?

As stated in https://github.com/jomjol/AI-on-the-edge-device/issues/1305#issuecomment-1309118947 this should be fixed in rolling

**Comment by jonathan-schmieg** [comment]
I have the same issue - sorry for my potentially dumb question: what does "in rolling" mean?

**Comment by stevebeer** [comment]
> I have the same issue - sorry for my potentially dumb question: what does "in rolling" mean?

'Rolling' is the name of the branch in GitHub. 
Stable is the normal branch where the released versions come from.

I tried both.

**Comment by jomjol** [comment]
@stevebeer Can you please remove all comment also in [MakeImage:
```
[MakeImage]
;LogImageLocation = /log/source
WaitBeforeTakingPicture = 5
;LogfileRetentionInDays = 15
```

I still have no glue, where the problem is coming from.

**Comment by jonathan-schmieg** [comment]
do I have to compile rolling myself or is there a sneak preview somewhere?

**Comment by caco3** [comment]
You can get it from  https://github.com/jomjol/AI-on-the-edge-device/actions?query=branch%3Arolling

**Comment by Slider0007** [comment]
This issue was fixed with #1377 and is already included in the latest releases. Please reopen, if issue still exists.
